core/src/main/scala/akka/persistence/cassandra/journal/TagWriters.scala

```diff
@@ -329,14 +343,15 @@ import scala.util.Try
 
   }
 
-  private def forwardTagWrite(tw: TagWrite): Unit = {
+  private def forwardTagWrite(tw: TagWrite): Future[Done] = {
     if (tw.actorRunning && !currentPersistentActors.contains(tw.serialised.head.persistenceId)) {
       log.warning(
         "received TagWrite but actor not active (dropping, will be resolved when actor restarts): [{}]",
         tw.serialised.head.persistenceId)
+      Future.successful(Done)
     } else {
       updatePendingScanning(tw.serialised)
-      tellTagActor(tw.tag, tw, sender())
```

Comment:
do we still need `tellTagActor` or can be simplify by changing to askTagActor for the remaining usage of tellTagActor?

Reply:
Nope! Seems we don't ever send a TagFlush to TagWriters anyhow